src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -86,10 +88,14 @@ public ICredentialProvider AcquireCredentials(Dictionary<String, String> args, b
             {
                 // get from user
                 var consoleWizard = HostContext.GetService<IConsoleWizard>();
+                string serverUrl = args[CliArgs.Url];
+                //on premise defaults to negotiate authentication (Kerberos with fallback to NTLM)
+                //hosted defaults to PAT authentication
+                string defaultAuth = serverUrl.Contains("visualstudio.com") || serverUrl.Contains("tfsallin.net") ? "PAT" : "Negotiate";
```

Comment:
Consider using `IndexOf(string value, System.StringComparison comparisonType)` instead of Contains to user experience confusion over casing subtleties.